packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -438,12 +452,10 @@ export class AzureCommunicationCallWithChatAdapter implements CallWithChatAdapte
   }
   /** Load previous Chat messages. */
   public async loadPreviousChatMessages(messagesToLoad: number): Promise<boolean> {
-    if (this.chatAdapter) {
-      return this.chatAdapter.loadPreviousChatMessages(messagesToLoad);
-    } else {
-      const chatAdapter = await this.chatAdapterPromise;
-      return chatAdapter.loadPreviousChatMessages(messagesToLoad);
+    if (!this.chatAdapter) {
+      throw new Error('ChatAdapter is not initialized');
```

Comment:
Avoid the term `adapter` in the error (developers aren't exposed to the fact that a callwithchatadapter is driven by two internal adapters)
```suggestion
      throw new Error('Chat is not initialized');
```